backend/src/main/java/com/bakdata/conquery/models/events/stores/primitive/StringStoreString.java

```diff
@@ -67,8 +67,8 @@ public int getLines() {
 	}
 
 	@Override
-	public <T extends ColumnStore> T createDescription() {
-		return null;
+	public StringStoreString createDescription() {
```

Comment:
In dem PR geht es dann vor allem um diese Funktion?

Reply:
Jep, ich gehe davon aus, dass hier der return value einfach vergessen wurde